src/main/resources/openapi/transcriptions.yaml

```diff
@@ -46,6 +46,66 @@ paths:
             schema:
               $ref: '#/components/schemas/TranscriptionRequestDetails'
   /transcriptions/{transcription_id}:
+    get:
+      tags:
+        - Transcription
+      summary: Gets details of a transcription
+      parameters:
+        - in: path
+          name: transcription_id
+          schema:
+            type: integer
+          description: "transcription_id is the internal id of the transcription."
+          required: true
+      operationId: getTranscription
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TranscriptionResponse'
+        '403':
+          description: Forbidden Error
+          content:
+            application/json+problem:
+              schema:
+                allOf:
+                  - $ref: './problem.yaml'
+                  - type: object
+                    required:
+                      - type
+                    properties:
+                      type:
+                        $ref: '#/components/schemas/TranscriptionAuthorisation403ErrorCode'
+              example:
+                type: "AUTHORISATION_100"
+                title: "User is not authorised for the associated courthouse"
+                status: 403
+        '404':
+          description: Not Found Error
+          content:
+            application/json+problem:
+              schema:
+                allOf:
+                  - $ref: './problem.yaml'
+                  - type: object
+                    required:
+                      - type
+                    properties:
+                      type:
+                        $ref: '#/components/schemas/UpdateTranscription404ErrorCode'
+              example:
+                type: "TRANSCRIPTION_101"
+                title: "The requested transcription cannot be found"
+                status: 404
+        '500':
```

Comment:
I don't think we should expect to see a 500 response normally tbh

Reply:
Error 500 has now been removed from DMP-1329